lib/icinga/scheduleddowntime.cpp

```diff
@@ -236,11 +249,16 @@ void ScheduledDowntime::CreateNextDowntime()
 	}
 
 	double minEnd = 0;
+	auto downtimeOptionsHash (HashDowntimeOptions());
 
 	for (const Downtime::Ptr& downtime : GetCheckable()->GetDowntimes()) {
 		if (downtime->GetScheduledBy() != GetName())
 			continue;
 
+		auto configOwnerHash (downtime->GetConfigOwnerHash());
+		if (!configOwnerHash.IsEmpty() && configOwnerHash != downtimeOptionsHash)
+			continue;
```

Comment:
So downtimes that don't have a config owner hash set will still be considered for `minEnd`. Therefore, I think if you add a range that's earlier than the next one already scheduled, it won't reschedule. Thus, probably won't fix #8630 in the current state.

Reply:
The fact you mentioned in the review above already makes this construct temporarily somewhat broken. Users will have to let it run for a while (until the old DTs vanish).